Makefile

```diff
@@ -42,4 +35,4 @@ clean:
 test : test1
 
 test1:
-	go test
+	go test
```

Comment:
For testing, please use `go test -v` or even better `go test -v -race -bench=.` if you write bench tests. The race flag will test go code for racing conditions too.